docs/resources/registry.md

```diff
@@ -0,0 +1,47 @@
+# Resource: harbor_registry
+
+
+
+## Example Usage
+
+```hcl
+    resource "harbor_registry" "main" {
+		provider_name = "docker-hub"
```

Comment:
Make sure the indents are the same, it looks a bit more clean

Reply:
I've fixed formatting